docs/content/theme/_index.md

```diff
@@ -1,42 +1,68 @@
-	</pfe-cta>
+Often components will make decisions about how to best utilize the colors from the palette. A standard call-to-action component (`pfe-cta`), for example, uses the standard link color in its default state (`--pfe-theme--color--ui-link`), but if you set the `priority` attribute value to `primary`, the CTA will make use of the accent color from the palette for the background color instead.
+
+```html
+<pfe-cta priority="primary">
+    <a href="#">Primary</a>
+</pfe-cta>
+```
+
+<br/>
+<pfe-cta priority="primary">
+    <a href="#">Primary</a>
+</pfe-cta>
 
+The `priority="primary"` variation of the `pfe-cta` makes use of the `ui-accent` color for the background because that link should be the brightest, boldest, most attention-grabbing item on the page.
 
-This is because the accent color should be the brightest and boldest, and the primary call-to-action should be the most attention-grabbing item on the page.
+By default, the `pfe-cta` styles call a particular variable stack comprised of a local variable (such as `--pfe-cta--BackgroundColor`), a theme variable (such as `--pfe-theme--color--ui-accent`), and a fallback value (`#06c`).  When putting a component on your page with no additional code, you will see your button using the fallback value, `#06c`.
 
+But what happens if you don't want the standard `ui-accent` on your page to be `#06c`?
 
-Additionally, each component comes equipped to adjust its colors depending on where it's placed on the page. For example, should you need to put a default CTA (which is blue) on a dark blue card, the CTA will need to adapt. You can do this by informing the component of its context (on a dark background) by giving the `on` attribute the value of `dark`.
+Your site can customize the value of any of the available theme assets!  You can change your `ui-accent` color for your entire site to be `#e00` by adding:
 
-    <pfe-card color="dark">
-	    <pfe-cta color="base">
-	    	<a href="#">Default</a>
-	    </pfe-cta>
-	</pfe-card>
+```css
+:root {
+	--pfe-theme--color--ui-accent: #e00;
+}
+```
 
-Should you need to deviate from this color usage, and set your primary CTA to use the complement color from the palette, you may also pass a value of `complement` into the `pfe-color`  attribute, like this:
+(You don't have to use `:root`, it's just a common example in documentation related to CSS variables.)
 
-	<pfe-cta priority="primary" color="complement">
-		<a href="#">Primary</a>
+Any other component making use of that same `ui-accent` value will inherit your customization creating a consistent experience for your users.
+
+## Context
+
+Each component comes equipped to adjust its colors depending on the context inside which it exists. For example, should you need to put a default CTA (which is blue) on a dark blue card, the CTA will need to adapt so that the text remain readable. You can do this by informing the component of its context by giving the `context` attribute the value of `dark`.
```

Comment:
```suggestion
Each component comes equipped to adjust its colors depending on the context inside which it exists. For example, should you need to put a default CTA (which is blue) on a dark blue card, the CTA will need to adapt so that the text remain readable. The card component therefore needs to pass down a `context` to the CTA, which it does using the `on="dark"` attribute.  This is programmatically applied by JavaScript that runs from the base class, `PFElement` (you don't have to do a thing!).  To manually override a component's context, you can do this by setting the `context="dark"` attribute.
```